arcade/sprite_list/sprite_list.py

```diff
@@ -156,7 +156,7 @@ def __init__(
 
         self._spatial_hash_cell_size = spatial_hash_cell_size
         self.spatial_hash: Optional[SpatialHash] = None
-        if use_spatial_hash is True:
```

Comment:
@einarf In re to your previous comment. The original version of the condition here evaluates to `False` for both `None` and `False`. My proposed change simplifies that, both making the code shorter as well as using a the boolean value directly. My assumption is that when `use_spatial_hash` defaults to `False`, this will fall back to checking collision of an object with every other object.

Reply:
This is reasonable 👍